src/main/resources/templates/db_scripts/schema.json

```diff
@@ -723,6 +723,11 @@
       "run": "after",
       "snippetPath": "alterHridSequences.sql",
       "fromModuleVersion": "18.2.0"
+    },
+    {
+      "run": "after",
+      "snippetPath": "migrateItemCopyNumberToSingleValue.sql",
+      "fromModuleVersion": "18.2.0"
```

Comment:
The next release is scheduled to be 18.3.0, should this be 18.3.0?

Reply:
Actually, this change introduces a breaking compatibility change, which I think means the implementation version should change to 19.0.0

Reply:
Ok, make sense. Should I change the implementation version as well?

Reply:
Yes, please change the implementation version to 19.0.0